### PR TITLE
8 missing sweetest for external id

### PR DESCRIPTION
**Pull request by nphias**

### CODE REVIEW

File: crates/holoom_dna_tests/src/tests/username_registry/external_id_attestation.rs [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+            },
+        )
+        .await;
+
+    assert!(result.is_err());
+}
+
+
+
+//this test anticipates future code changes that chnage tht behaviour
```
Comment (8e8b2c) [inline crates/holoom_dna_tests/src/tests/username_registry/external_id_attestation.rs:46]: Spelling! I know I've asked you delete this test anyway, but this seems like a good opportunity to remind you about:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

**Review by 8e8b2c — CHANGES_REQUESTED** [review]
Nice, just a few changes please.

**Comment by nphias** [comment]
done

**Review by 8e8b2c — APPROVED** [review]
Thanks!

**Pull request merged**